docs/UserGuide.md

```diff
@@ -212,6 +213,25 @@ Notes:
 **Sample Usage:**
 * `find Friend NUS`
     * returns people tagged with both `Friend` and `NUS`
```

Comment:
Might want to combine these 2 return statements since there should only be one outcome